source/src/microapp/cs_MicroappInterruptHandler.cpp

```diff
@@ -544,6 +549,23 @@ void MicroappInterruptHandler::onBlePeripheralNotififyDone(uint16_t connectionHa
 	MicroappController::getInstance().generateSoftInterrupt(CS_MICROAPP_SDK_TYPE_BLE, CS_MICROAPP_SDK_BLE_PERIPHERAL);
 }
 
+void MicroappInterruptHandler::onAssetAccepted(AssetAcceptedEvent& event) {
```

Comment:
If you look at `AssetFiltering::handleAcceptedAsset` you'll see that there are different output formats. You may want to use this, or simply always add the MAC address to the data towards the microapp (handy for connecting?). Maybe even add the whole advertising data, not sure if that fits in the buffer though.

Reply:
And maybe also add the profile ID ?